pmlc/conversion/linalg_to_pxa/test/conv.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: pmlc-opt -convert-linalg-to-pxa %s | FileCheck %s
+
+module  {
+  func @test_conv(%arg0: memref<128x128x3xf32, offset: ?, strides: [?, ?, 1]>, %arg1: memref<3x3x3xf32, offset: ?, strides: [?, ?, 1]>, %arg2: memref<?x?x?xf32, offset: ?, strides: [?, ?, 1]>) {
+    linalg.conv(%arg0, %arg1, %arg2) {strides = [2]}: memref<128x128x3xf32, offset: ?, strides: [?, ?, 1]>, memref<3x3x3xf32, offset: ?, strides: [?, ?, 1]>, memref<?x?x?xf32, offset: ?, strides: [?, ?, 1]>
```

Comment:
Are we required to use `?` for offset and strides? Can we make this work when we have no layout?

Reply:
I just copied the test case from Linalg's test cases. I've removed ? in the case.